crates/viewer/re_view_time_series/src/line_visualizer_system.rs

```diff
@@ -233,11 +222,37 @@ impl SeriesLineSystem {
                     .map(|index| (index, ()))
             };
 
+            // Determine how many lines we have.
+            // TODO(andreas): We should determine this only once and cache the result.
+            // As data comes in we can validate that the number of series is consistent.
+            // Keep in mind clears here.
+            let num_series = all_scalar_chunks
+                .iter()
+                .next()
+                .and_then(|chunk| chunk.iter_slices::<f64>(Scalar::name()).next())
+                .map(|slice| slice.len())
+                .unwrap_or(1);
+            if num_series == 0 {
+                re_log::warn_once!("Empty scalar array found for {entity_path:?}");
+                return;
```

Comment:
this is actually a bit fishy. What if the series starts with an empty array for whatever reason. Should at least search until we find one that isn't empty.